src/PaymentPart/Output/Element/Placeholder.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Sprain\SwissQrBill\PaymentPart\Output\Element;
+
+class Placeholder implements OutputElementInterface
+{
+    const PLACEHOLDER_TYPE_PAYABLE_BY = [
+        'type' => 'placeholder_payable_by',
+        'file' => __DIR__ . '/../../../../assets/marks_65x25mm.svg',
+        'width' => 65,
+        'height' => 25
+    ];
+
+    const PLACEHOLDER_TYPE_PAYABLE_BY_RECEIPT = [
+        'type' => 'placeholder_payable_by_receipt',
+        'file' => __DIR__ . '/../../../../assets/marks_52x20mm.svg',
+        'width' => 52,
+        'height' => 20
+    ];
+
+    const PLACEHOLDER_TYPE_AMOUNT = [
+        'type' => 'placeholder_amount',
+        'file' => __DIR__ . '/../../../../assets/marks_40x15mm.svg',
+        'width' => 40,
+        'height' => 15
+    ];
+
+    const PLACEHOLDER_TYPE_AMOUNT_RECEIPT = [
+        'type' => 'placeholder_amount_receipt',
+        'file' => __DIR__ . '/../../../../assets/marks_30x10mm.svg',
+        'width' => 30,
+        'height' => 10
+    ];
+
+    /** @var string */
+    private $type;
+
+    /** @var string */
+    private $file;
+
+    /** @var int */
+    private $width;
+
+    /** @var int */
+    private $height;
+
+    public static function create(array $type)
```

Comment:
I wonder if we can create private constructors for all classes with named constructors, then we'll be able to use strict typehinting (without nullable typehint) for the getters - we'll be sure that it will be filled in.

Also there is missing typehint for all the static `create` methods